src/cascadia/TerminalSettingsModel/FontConfig.idl

```diff
@@ -17,6 +17,8 @@ namespace Microsoft.Terminal.Settings.Model
         INHERITABLE_FONT_SETTING(Int32, FontSize);
         INHERITABLE_FONT_SETTING(Windows.UI.Text.FontWeight, FontWeight);
 
+        // The INHERITABLE_FONT_SETTING macro doesn't like Windows.Foundation.Collections.IMap<String, UInt32>

```

Comment:
Uhhhh. I feel like either @DHowett or @lhecker would have an idea of how to fix the macro...